packages/relay/src/lib/services/ethService/contractService/ContractService.ts

```diff
@@ -821,13 +821,6 @@ export class ContractService implements IContractService {
     blockNumberOrTag: string | null,
     requestDetails: RequestDetails,
   ): Promise<string | JsonRpcError> {
-    // ETH_CALL_DEFAULT_TO_CONSENSUS_NODE = false enables the use of Mirror node
-    const shouldDefaultToConsensus = ConfigService.get('ETH_CALL_DEFAULT_TO_CONSENSUS_NODE');
-
-    if (shouldDefaultToConsensus) {
-      return await this.callConsensusNode(call, gas, requestDetails);
-    }
-
     return await this.callMirrorNode(call, gas, call.value, blockNumberOrTag, requestDetails);
```

Comment:
given this is the only method call left, maybe we can inline this function and remove the `routeAndExecuteCall` method.